doc_src/en/OmegaT4_EditorsPanes.xml

```diff
@@ -214,9 +214,12 @@
     <guimenuitem>Ctrl+R</guimenuitem> to replace the target field with the match or use
     <guimenuitem>Ctrl+I </guimenuitem> to insert it at the cursor position.</para>
 
-    <para>The matching percentage is roughly equivalent to taking the number
-    of common words in the matched and the matching segment and dividing by
-    the number of words in the longer of the two. The selected fuzzy match is
+    <para>three matching percentages are in order: the percentage calculated with
```

Comment:
```suggestion
    <para>The three matching percentages are in order: the percentage calculated with
```

Reply:
This is definitely an improvement. Things that could be improved more:

1. Formatting: This would probably be more readable as a list of three items
2. Jargon: Is stemming explained anywhere in the manual? I can't remember. It's not explained in the UI, but that's acceptable due to limited space. Here, we could even just link to e.g. https://en.wikipedia.org/wiki/Stemming
3. Implication: It would be nice to clarify that while you can change the sort order of the five fuzzy matches, you can't change which percentage is used to select the five fuzzy matches.

Bonus: You could reflow the second paragraph. I do this in Emacs with `M-q`.

Reply:
Ok, I'll rewrite and resubmit. Thank you for the remarks. Re. emacs, the reflowing did not use the same width as the xml so I gave it up. I'll do that too.

Reply:
You can make changes as additional commits and just push them to this branch. No need to close the PR.

I think I might have found that Emacs's reflow didn't match the original, but I don't consider that a problem. To keep the diff from getting crazy, though, I would reflow only the paragraphs you actually touch.